code/game/gamemodes/changeling/changeling.dm

```diff
@@ -204,10 +204,13 @@ var/list/possible_changeling_IDs = list("Alpha","Beta","Gamma","Delta","Epsilon"
 						feedback_add_details("changeling_objective","[objective.type]|FAIL")
 						changelingwin = 0
 					count++
+					karma_reward = count - 1
 
 			if(changelingwin)
 				text += "<br><font color='green'><B>The changeling was successful!</B></font>"
 				feedback_add_details("changeling_success","SUCCESS")
+				sql_report_objective_karma(changeling.key, karma_reward)
+				to_chat(world, "<b>[changeling.key] got [karma_reward] karma points for completing objectives!</b>")

```

Comment:
Awarding karma for completing objectives. I'd like to see how this actually works for them